t/docker/script/openSUSE:Leap:15.2:WSL.sh

```diff
@@ -4,18 +4,18 @@ set -ex
 source lib/common.sh
 
 # mock OBS backend folder
-mkdir -p /mockOBS/Virtualization:WSL/openSUSE_Leap_15.2/x86_64/wsl-appx/
+mkdir -p /mockOBS/openSUSE:Leap:15.2:WSL/openSUSE_Leap_15.2/x86_64/wsl-appx/
 # create just empty files as at this point we are not interested about proper binaries
-touch /mockOBS/Virtualization:WSL/openSUSE_Leap_15.2/x86_64/wsl-appx/openSUSE-x64-Build20191128.7.9.appx
-touch /mockOBS/Virtualization:WSL/openSUSE_Leap_15.2/x86_64/wsl-appx/openSUSE-x64-Build20191128.7.9.appx.sha256
+touch /mockOBS/openSUSE:Leap:15.2:WSL/openSUSE_Leap_15.2/x86_64/wsl-appx/openSUSE-x64-Build20191128.7.9.appx
+touch /mockOBS/openSUSE:Leap:15.2:WSL/openSUSE_Leap_15.2/x86_64/wsl-appx/openSUSE-x64-Build20191128.7.9.appx.sha256
 chown -R "$dbuser" /mockOBS
 
-prepare_project Virtualization:WSL -other
+prepare_project openSUSE:Leap:15.2:WSL -other
```

Comment:
@mloviska sorry it is not obvious and not documented properly - but here '-other' means that we try to sync from `obspublish-other` as it was specified in old xml . (this is how projects partitioned inside obs).
The new xml does sync from `obspublish`, so could you please just remove `-other` and leave only project name.

Reply:
No worries. I haven't gone so far with the code to check what actually it means.
Nice catch, thanks for review!